src/components/ebay-menu/template.marko

```diff
@@ -0,0 +1,53 @@
+<script marko-init>
+    var processHtmlAttributes = require("../../common/html-attributes");
+</script>
+
+<var isFake=(data.type === 'fake')/>
+<var isRadio=(data.type === 'radio')/>
+<var isCheckbox=(data.type === 'checkbox')/>
+<var isNotCheckable=(data.type !== "checkbox" && data.type !== "radio")/>
+<var baseClass = (data.classPrefix || (isFake ? 'fake-menu' : 'menu'))/>
+
+<span
+    w-bind
+    class=[
+        (data.classPrefix ? "${baseClass}__menu": (isFake ? 'fake-menu' : 'menu')),
+        data.class
+    ]
+    style=data.style
+    ${processHtmlAttributes(data)}>
+    <div
+        w-id="menu"
+        role=(!isFake && 'menu')
+        class=[
+            "${baseClass}__items",
+            (data.reverse && "${baseClass}__items--reverse"),
+            (data.fixWidth && "${baseClass}__items--fix-width")
+        ]>
+        <for(item in data.items)>
+            <var itemRole = (isRadio ? 'menuitemradio' : (isCheckbox ? 'menuitemcheckbox' : !isFake && 'menuitem'))/>
```

Comment:
I think this could end up with an empty itemRole, if isRadio is false and isCheckbox is false, and isFake is true.  Will that cause any problems?

Reply:
That's expected. That specific case it should not have a role. (E.g. a fake menu with links should not have `role` on any of the links or buttons.)